bidict/_bidict.py

```diff
@@ -73,10 +79,10 @@ def clear(self):
         self._inv.clear()
 
     def pop(self, key, *args):
-        """Like :py:meth:`dict.pop`, keeping bidirectionality intact."""
-        ln = len(args) + 1
-        if ln > 2:
-            raise TypeError('pop expected at most 2 arguments, got %d' % ln)
+        """Like :py:meth:`dict.pop`, maintaining bidirectionality."""
+        l = len(args) + 1
```

Comment:
I actually use `l` as a variable name often, and have rarely had a problem, but the font I'm seeing for code reviews on Github (Consolas, 12px) is one of those where `l` looks almost exactly like `1`, which is why I flagged it up.